src/Controller/CompanyController.php

```diff
@@ -11,68 +9,136 @@
 use Symfony\Component\Form\Extension\Core\Type\EmailType;
```

Comment:
This class is not used , remove it. Remove all classes and variables that are not used